LEGO1/define.h

```diff
@@ -1,6 +1,8 @@
 #ifndef DEFINE_H
 #define DEFINE_H
 
+extern const char *g_str_enable;
+extern const char *g_str_disable;
```

Comment:
I think we should only put strings here that we know are used across multiple compilation units. It seems those are only used in a single function?

Reply:
Good point, I'll switch this around. (The string "disable" _is_ used in LegoGameState too, but it is referenced directly rather than indirectly as in LegoFullScreenMovie, so it's not a global variable.)